docs/src/api/models.md

```diff
@@ -0,0 +1,30 @@
+```@meta
+CurrentModule = Metalhead
+```
+
+# Models
+
+The API reference for available models in `Metalhead.jl`.
+
+```@docs
+VGG
+ResNet
+WideResNet
+GoogLeNet
+Inception-v3
```

Comment:
The Inception family doesn't have a hyphen in the model names, so this should be

```suggestion
Inceptionv3
```

and so on